Packs/PiHole/Integrations/PiHole/PiHole.yml

```diff
@@ -0,0 +1,220 @@
+commonfields:
+  id: PiHole
+  version: -1
+name: PiHole
+display: PiHole
+category: IT Services
+description: This is an integration to PiHole
```

Comment:
please improve description to describe what this integration can be used for

Reply:
Added more details into the description